styles/rentry/catppuccin.user.css

```diff
@@ -0,0 +1,526 @@
+    }
+    .form-control,
+    .tab-content,
+    textarea {
+      background: @base !important;
+      color: @text;
+    }
+    .input-success {
+      box-shadow: inset 4px 0 @green !important;
+    }
+    .input-error {
+      box-shadow: @red 4px 0px inset !important;
+    }
+    .input-warning {
+      box-shadow: @yellow 4px 0px inset !important;
+    }
+    .btn {
+      background: @base !important;
+      color: @text !important;
+    }
+    .btn:hover {
+      background: @surface0 !important;
+      color: @text !important;
+    }
+    .clipboard {
+      color: @text;
+      background: none;
+    }
+    #progressBar {
+      background-color: @blue !important;
+    }
+    .full-contrast:not(.active) path {
+      fill: @overlay2 !important;
+    }
+    .full-contrast.active path {
+      fill: @peach !important;
+    }
+    .edit-code {
+      background: @mantle;
+      color: @text !important;
+    }
+    .cb-tooltiptext {
+      color: @text !important;
+      background: @surface0 !important;
+    }
+    .cb-tooltiptext-bottom::after {
+      border-color: transparent transparent @surface0 transparent;
+    }
+    .headerlink {
+      color: @overlay1 !important;
+    }
+    #submitButton {
+      background: @mantle !important;
+    }
+    // claim-guide page
+    .color-change {
+      &[style="color:grey"] {
+        color: @overlay2 !important;
+      }
+      &[style="color:#039205"] {
+        color: @green !important;
+      }
+    }
+    // delete modal
+    .modal-content {
+      background: @crust;
+      color: @text;
+    }
+    .modal-body {
+      color: @overlay2;
+    }
+    .modal-header {
+      color: @text;
+    }
+    .modal-header,
+    .modal-footer {
+      border-color: @surface1 !important;
+    }
+    #deleteButton {
+      color: @red !important;
+    }
+    // footer
+    a {
+      color: @blue;
+    }
+    a:hover {
+      color: @sky !important;
+    }
+    .text-muted {
+      color: @overlay2 !important;
+    }
+    .text-primary {
+      color: @blue !important;
+    }
+    .text-success {
+      color: @green !important;
+    }
+    .text-warning {
+      color: @yellow !important;
+    }
+    .text-danger {
+      color: @red !important;
+    }
+    // editor
+    .CodeMirror-line::selection,
+    .CodeMirror-line > span::selection,
+    .CodeMirror-line > span > span::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+    .cm-formatting-admonition {
+      color: @yellow;
+    }
+    .cm-mark {
+      color: @yellow;
+    }
+    .cm-link {
+      color: @blue;
+    }
+    .cm-url {
+      color: @sky;
+    }
+    .cm-formatting-toc {
+      color: @mauve;
+    }
+    .cm-variable-2 {
+      color: @green;
+    }
+    .cm-keyword {
+      color: @pink;
+    }
+    .cm-meta {
+      color: @subtext0;
+    }
+    .cm-comment {
+      color: @pink !important;
+    }
+    .cm-hr {
+      color: @surface2;
+    }
+    // markdown
+    .cm-header {
+      color: @text;
+    }
```

Comment:
Same thing with my other syntax highlighting comment, make sure it actually resembles our syntax highlighting ports.

Reply:
the [style guide](https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md) is focused on code based syntax highlighting (terminals, code editors) but these are colors for (afaik) rentry specific syntax (i don't know other languages that support something specific like  ==marks== or how i am supposed to theme a [TOC] element). i guess the closest thing would be markdown but this seems like it's own thing

![image](https://github.com/user-attachments/assets/4cfa4489-d8d1-4b0b-8277-a755bb170277)
![image](https://github.com/user-attachments/assets/7fcc9548-cb55-4ddf-b722-82bef804d976)
![image](https://github.com/user-attachments/assets/8b4888a0-e344-488f-8a19-4594da301243)